pkgs/by-name/fo/formatjson5/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "formatjson5";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "json5format";
+    # Not tagged, see Cargo.toml.
+    rev = "056829990bab4ddc78c65a0b45215708c91b8628";
+    hash = "sha256-35ftRr0XEU1AC98MiP8Q6dWIChPHcATkqPYfoVzZRmY=";
```

Comment:
I have tried to build it via nixpkgs-review, but it failed with  hash mismatch:

```
error: hash mismatch in fixed-output derivation '/nix/store/p35c72i9bk1v98q5yra7i2hy5vp06akn-source.drv':
         specified: sha256-35ftRr0XEU1AC98MiP8Q6dWIChPHcATkqPYfoVzZRmY=
            got:    sha256-Lredw/Fez+2U2++ShZcKTFCv8Qpai9YUvqvpGjG5W0o=
error: 1 dependencies of derivation '/nix/store/wcnizz1i4jq8jhybyj1afzw0p3lf4kw0-formatjson5-0.2.6-vendor.tar.gz.drv' failed to build
error: 2 dependencies of derivation '/nix/store/j2hx1g8mai2b747pbdrsikmqrn0dlifr-formatjson5-0.2.6.drv' failed to build
error: 1 dependencies of derivation '/nix/store/a9kipv2w1n00p7m0jzs4vi3jzg7lcgch-review-shell.drv' failed to build
```

Reply:
Thanks, should be fixed now. I moved the package to another dir and the build was still cached.